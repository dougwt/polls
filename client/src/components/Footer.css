footer,
.page-footer {
  background-color: #06bad6;
  line-height: 1.8rem;
}

.page-footer a:hover {
  color: #ef476f !important;
}

.page-footer .nav ul {
  font-family: 'Raleway', cursive;
  font-size: 0.9rem;
}

.footer-copyright a {
  color: #ffd166;
  font-weight: bold;
}

.footer-copyright .heart {
  color: #ef476f;
}
