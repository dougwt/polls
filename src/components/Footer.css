footer,
.page-footer {
  background-color: #06bad6;
  line-height: 1.8rem;
}

.page-footer p {
  font-size: 0.9rem;
}

.page-footer a:hover {
  color: #ef476f !important;
}

.page-footer .nav ul {
  font-family: 'Raleway', cursive;
  font-size: 0.9rem;
}

.page-footer a,
.page-footer a:visited {
  color: #f9cd69;
}

.footer-copyright a {
  color: #ffd166;
  font-weight: bold;
}

.footer-copyright .heart {
  color: #ef476f;
}
