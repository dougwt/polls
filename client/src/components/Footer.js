import React from 'react';
import './Footer.css';

const Footer = () => {
  return (
    <footer className="page-footer">
      <div className="container">
        <div className="row">
          <div className="col l9 s12">
            <h5 className="white-text">Simple Polls</h5>
            <p className="grey-text text-lighten-4">
              Lorem ipsum dolor sit amet, consectetur adipisicing elit.
              Dignissimos earum veritatis numquam beatae harum fuga culpa itaque
              quaerat minus omnis accusantium ipsam aperiam esse mollitia,
              tempore porro, a quasi libero!
            </p>
          </div>
          <div className="col l3 s12 nav">
            <h5 className="white-text">Site Navigation</h5>
            <ul>
              <li>
                <a className="white-text" href="#!">
                  Homepage
                </a>
              </li>
              <li>
                <a className="white-text" href="#!">
                  Dashboard
                </a>
              </li>
              <li>
                <a className="white-text" href="#!">
                  Create a Poll
                </a>
              </li>
              <li>
                <a className="white-text" href="#!">
                  GitHub Repo
                </a>
              </li>
            </ul>
          </div>
        </div>
      </div>

      <div className="footer-copyright">
        <div className="container">
          Made with <span className="heart">&#9829;</span> by{' '}
          <a className="text-lighten-3" href="https://dougwt.com">
            @dougwt
          </a>
        </div>
      </div>
    </footer>
  );
};

export default Footer;
