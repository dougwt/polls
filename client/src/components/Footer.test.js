import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

describe('Footer', () => {
  const footer = shallow(<Footer />);

  it('renders properly', () => {
    expect(footer).toMatchSnapshot();
  });

  it('contains a copyright', () => {
    expect(footer.find('.footer-copyright').exists()).toBe(true);
  });
});
