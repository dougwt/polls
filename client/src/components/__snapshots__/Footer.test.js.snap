// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="container"
>
          <div
                    className="row"
          >
                    <div>
                              <p>
                                        Footer
                              </p>
                    </div>
          </div>
</div>,
        <div
          className="footer-copyright"
>
          <div
                    className="container"
          >
                    © 2014 Copyright Text
                    <a
                              className="grey-text text-lighten-4 right"
                              href="#!"
                    >
                              More Links
                    </a>
          </div>
</div>,
      ],
      "className": "page-footer",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="row"
>
            <div>
                        <p>
                                    Footer
                        </p>
            </div>
</div>,
          "className": "container",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div>
              <p>
                            Footer
              </p>
</div>,
            "className": "row",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <p>
                Footer
</p>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Footer",
              },
              "ref": null,
              "rendered": "Footer",
              "type": "p",
            },
            "type": "div",
          },
          "type": "div",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="container"
>
            © 2014 Copyright Text
            <a
                        className="grey-text text-lighten-4 right"
                        href="#!"
            >
                        More Links
            </a>
</div>,
          "className": "footer-copyright",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "© 2014 Copyright Text",
              <a
                className="grey-text text-lighten-4 right"
                href="#!"
>
                More Links
</a>,
            ],
            "className": "container",
          },
          "ref": null,
          "rendered": Array [
            "© 2014 Copyright Text",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "More Links",
                "className": "grey-text text-lighten-4 right",
                "href": "#!",
              },
              "ref": null,
              "rendered": "More Links",
              "type": "a",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
    ],
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="container"
>
            <div
                        className="row"
            >
                        <div>
                                    <p>
                                                Footer
                                    </p>
                        </div>
            </div>
</div>,
          <div
            className="footer-copyright"
>
            <div
                        className="container"
            >
                        © 2014 Copyright Text
                        <a
                                    className="grey-text text-lighten-4 right"
                                    href="#!"
                        >
                                    More Links
                        </a>
            </div>
</div>,
        ],
        "className": "page-footer",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="row"
>
              <div>
                            <p>
                                          Footer
                            </p>
              </div>
</div>,
            "className": "container",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div>
                <p>
                                Footer
                </p>
</div>,
              "className": "row",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <p>
                  Footer
</p>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Footer",
                },
                "ref": null,
                "rendered": "Footer",
                "type": "p",
              },
              "type": "div",
            },
            "type": "div",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="container"
>
              © 2014 Copyright Text
              <a
                            className="grey-text text-lighten-4 right"
                            href="#!"
              >
                            More Links
              </a>
</div>,
            "className": "footer-copyright",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "© 2014 Copyright Text",
                <a
                  className="grey-text text-lighten-4 right"
                  href="#!"
>
                  More Links
</a>,
              ],
              "className": "container",
            },
            "ref": null,
            "rendered": Array [
              "© 2014 Copyright Text",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "More Links",
                  "className": "grey-text text-lighten-4 right",
                  "href": "#!",
                },
                "ref": null,
                "rendered": "More Links",
                "type": "a",
              },
            ],
            "type": "div",
          },
          "type": "div",
        },
      ],
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
